testing/e2e/workload_test.go

```diff
@@ -65,8 +65,8 @@ func TestCreateFromGitWithAnnotations(t *testing.T) {
 					"--git-repo=https://github.com/sample-accelerators/spring-petclinic",
 					namespaceFlag,
 					"--type=web",
-					"--yes",
```

Comment:
shouldn't `--yes` skip the survey prompt ?

Reply:
yes but the survey prompt is being managed by `c.SurveyAnswer("y")` for this test